src/pages/OnboardingWork/index.native.tsx

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import BaseOnboardingWork from './BaseOnboardingWork';
 import type {OnboardingWorkProps} from './types';
 
-function OnboardingWork({...rest}: Omit<OnboardingWorkProps, 'shouldUseNativeStyles'>) {
```

Comment:
I was getting lint failure because of `withOnyx` so I migrated that. After that it started giving lint error for this line. So I solved it. Let me know if I should undo the changes for withOnyx hook